test/controllers/organizations/staff/fosterers_controller_test.rb

```diff
@@ -39,4 +40,49 @@ class Organizations::Staff::FosterersControllerTest < ActionDispatch::Integratio
       end
     end
   end
+
+  context "#edit" do
```

Comment:
There's separate test files for asserting authorization see `test/policies/organizations/fosterer_invitation_policy_test.rb`